### DISCUSSION TITLE
N8N module

### DISCUSSION THREAD

**Discussion by casualmatt** [directus/directus#11414] [Feature Requests]
Hello,

I'm a NoCoder, and i love to Directus and N8N, but i still have to do all the api call manually, because there is no official library (AKA node) for Directus in n8n. 

Or to be correct there is a [Pull-request](https://github.com/n8n-io/n8n/pull/2342) but is silent from weeks.

Anybody is interested in a deeper integration between the 2 software?

**Comment by rijkvanzanten** [comment]
Hi @casualmatt! We're very interested in no-code automation. As a matter of fact, we have a new feature in the works for Directus itself that I think you will _really_ enjoy in this space 😁

**Reply by casualmatt** [reply]
I didn't see anything in the [road-map](https://directus.io/road-map/). 

But yes I'm a big user of Bubble, but there platform is not providing what i expect especially in performances, and that is something that i can find in Directus, but it still miss a lot of features, Workflow or [SDK for Nuxt ](https://github.com/pnodev/nuxt-directus)(maybe with support for Nuxt3 soon? or official support). 

For now for workflow i used N8N but there license is not a real GNU, and the auth system is not integrated in Directus. 

Soo, yes ready to switch to a WF module on Directus, if you need any beta tester, let me know =D

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
